pkg/network/setup/netstat.go

```diff
@@ -66,16 +66,6 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		return nil
 	}
 
-	// This is needed to be backwards compatible with vmi's which have status interfaces
-	// with the name not being set
-	if len(domain.Spec.Devices.Interfaces) == 0 && len(vmi.Status.Interfaces) == 1 && vmi.Status.Interfaces[0].Name == "" {
```

Comment:
Now that I think about this a bit more, this is puzzling (to me at least). 

How can the VM be up n' running **but** the domain fail to report the interface whose connectivity was extended from the pod interface ?  ( or any, for all it matters)

Reply:
Are you trying to convince me that this code is no longer needed? :)

In theory, this state may exist, just because someone detached the interfaces from the domain under the hood.

IMO this is just a leftover dead code, but to be on the safe side, the reasoning should also include its irrelevance compared to its original intention.

Reply:
> Are you trying to convince me that this code is no longer needed? :)

0_o ... I'm ... not sure.

I think you're right . This code is not needed - at least I can't understand how this scenario can come to exist .

> 
> In theory, this state may exist, just because someone detached the interfaces from the domain under the hood.
> 
> IMO this is just a leftover dead code, but to be on the safe side, the reasoning should also include its irrelevance compared to its original intention.

Whatever you prefer.